[2025-02-22T20:45:01.378+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: coingecko_ingestion_dag.fetch_data manual__2025-02-22T20:44:57.024518+00:00 [queued]>
[2025-02-22T20:45:01.386+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: coingecko_ingestion_dag.fetch_data manual__2025-02-22T20:44:57.024518+00:00 [queued]>
[2025-02-22T20:45:01.387+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-22T20:45:01.402+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): fetch_data> on 2025-02-22 20:44:57.024518+00:00
[2025-02-22T20:45:01.406+0000] {standard_task_runner.py:57} INFO - Started process 41 to run task
[2025-02-22T20:45:01.414+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'coingecko_ingestion_dag', 'fetch_data', 'manual__2025-02-22T20:44:57.024518+00:00', '--job-id', '198', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmprnioy4_v']
[2025-02-22T20:45:01.418+0000] {standard_task_runner.py:85} INFO - Job 198: Subtask fetch_data
[2025-02-22T20:45:01.501+0000] {task_command.py:415} INFO - Running <TaskInstance: coingecko_ingestion_dag.fetch_data manual__2025-02-22T20:44:57.024518+00:00 [running]> on host 5fdfec269bbd
[2025-02-22T20:45:01.602+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='etudiant' AIRFLOW_CTX_DAG_ID='coingecko_ingestion_dag' AIRFLOW_CTX_TASK_ID='fetch_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-22T20:44:57.024518+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-22T20:44:57.024518+00:00'
[2025-02-22T20:45:02.250+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-22T20:45:02.263+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=coingecko_ingestion_dag, task_id=fetch_data, execution_date=20250222T204457, start_date=20250222T204501, end_date=20250222T204502
[2025-02-22T20:45:02.310+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-22T20:45:02.335+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
